ccmlib/cluster.py

```diff
@@ -17,7 +17,7 @@
 
 class Cluster(object):
 
-    def __init__(self, path, name, partitioner=None, install_dir=None, create_directory=True, version=None, verbose=False, snitch='org.apache.cassandra.locator.PropertyFileSnitch', **kwargs):
+    def __init__(self, path, name, partitioner=None, install_dir=None, create_directory=True, version=None, verbose=False, snitch='org.apache.cassandra.locator.PropertyFileSnitch', ccm_logger=None, ccm_log_level=None, **kwargs):
```

Comment:
why not just use `logging` module directly ?, to retain the print option in dtest ?

Reply:
i.e. something like
```python
logger = logging.getLogger("scylla.ccm")
logger.info("whatever")
```

and then on dtest configure what's  needed for it to be printed on stdout, when we need

Not only the cluster class has logging we care about, also other part ccm might have logs we want utility functions, the class download the version and such.